samples/readme.md

```diff
@@ -10,6 +10,4 @@ The samples available throughout the tree are:
 * IoTHub Service SDK samples are [here](../iothub_service_client/samples)
 * Serializer samples are  [here](../serializer/samples).  *Please read and understand the limitations and alternatives to the serializer described [here](../serializer/readme.md) before using the serializer.*
 * Provisioning Client SDK samples are [here](../provisioning_client/samples)
-* Provisioning Service SDK samples are [here](../provisioning_service_client/samples)
-
-You can find samples for end-to-end samples scenarios [here](solutions).
```

Comment:
Are these samples getting deprecated?

Reply:
Yes, that's the plan. The first ones are Service SDKs, which we don't actively maintain anymore. The second one is part of an old initiative to create end-to-end samples, but that has not been maintained for the last couple of years, so I'm killing it as well.

Reply:
Is there a plan to remove the code as well, or just remove links to it so anyone currently using it isn't broken?